src/components/Form/Form.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import './Form.scss';
+
+interface FormProps {
+  onChange: (newValues: {
+    step?: number;
+    frameSize?: number;
+    itemWidth?: number;
+    animationDuration?: number;
+    infinite?: boolean;
+  }) => void;
+  step: number;
+  frameSize: number;
+  itemWidth: number;
+  animationDuration: number;
+  infinite: boolean;
+}
+
+const Form: React.FC<FormProps> = ({
```

Comment:
Use a more descriptive name
```suggestion
const CarouselForm: React.FC<FormProps> = ({
```